charts/drax/templates/configmap-loki-tenant-override-config.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: loki-tenant-override-config
+  namespace: default
```

Use the usual mechanisms of helm here to guarantee unique names etc with multiple deployments in mind etc.

```suggestion
  name: {{ include accelleran.common.name (dict "top" .) }}-loki-tenant-override-config
```